### PR TITLE
Fix mobile issues in Shrine

### PR DESCRIPTION
**Pull request by blasten**
cc @frankiefu @keanulee

### CODE REVIEW

File: templates/shrine/src/shrine-item.html [hunk 1]
```diff
@@ -131,11 +113,6 @@
       [[item.storeName]]
     </div>
 
-    <div class="icons">
```
Comment (blasten) [inline templates/shrine/src/shrine-item.html]: this might be replaced by  `<template is="app-shared-layout">`

File: app-header/app-header.html [hunk 2]
```diff
@@ -444,7 +444,7 @@
        * @return {boolean}
        */
       isOnScreen: function() {
-        return this._height !== 0 && (this._top === 0 || this._top < this._height);
+        return this._height !== 0 && (this.fixed || this._top === 0 || this._top < this._height);
```
Comment (keanulee) [inline app-header/app-header.html]: Why is this needed? Shouldn't `this._top` be less than `this._height` if its fixed (and shouldn't `this._top === 0` not be needed as well)?

Comment (keanulee) [inline app-header/app-header.html]: Also, tests would be nice.

Comment (blasten) [inline app-header/app-header.html]: That is true..

Comment (blasten) [inline app-header/app-header.html]: test added

**Comment by keanulee** [comment]
LGTM

File: templates/shrine/src/shrine-simple-item.html [hunk 3]
```diff
@@ -20,9 +20,12 @@
         @apply(--layout-horizontal);
         @apply(--layout-center-center);
         @apply(--layout-wrap);
+        position: relative;
         text-align: center;
         width: 100%;
         height: 100%;
+        /* create a layer to only invalidate this layer for :hover */
+        transform: translateZ(0);
```
Comment (frankiefu) [inline templates/shrine/src/shrine-simple-item.html]: Should this move to `:host::after` since that's the one actually display the box-shadow on hover?

Comment (blasten) [inline templates/shrine/src/shrine-simple-item.html]: fixed @frankiefu

**Comment by frankiefu** [comment]
lgtm
